docs/registering-ticket-sales-multiple.md

```diff
@@ -0,0 +1,158 @@
+# Registering multiple ticket sales at once
+
+This mini-guide illustrates how to register UiTPAS discounted ticket sales for multiple tickets at once. It builds upon the [Registering ticket sales guide](/docs/registering-ticket-sales-group.md) which you need to read first to learn more about authentication, the work flow and registering events.
+
+### 1. Determine possible UiTPAS tariffs
+
+The register ticket sale request accepts an array of ticket sales. This array can contain multiple ticket sales for:
+
+*   one passholder for multiple events
+*   multiple passholders for one event
+*   or even multiple passholders for multiple events
+
+It is important to pass TicketSale objects with tariffs that are applicable to that passholder, for the given event. Do note that the register ticket sale request will fail for the complete set of ticket sales if one of those ticket sales would trigger an error.
+
+To determine the available tariffs, your application needs to use [GET /tariffs](/reference/UiTPAS.v2.json/paths/~1tariffs/get) for every passholder/event combination for which you need to register a ticket sale.
+
+Example request for passholder `0900000672312` and event `5a0967f9-cc06-4c3c-9206-30481a767434`:
+
+```http
+GET /tariffs/?eventId=5a0967f9-cc06-4c3c-9206-30481a767434&uitpasNumber=0900000672312&regularPrice=10 HTTP/1.1
+Host: https://api.uitpas.be
+Authorization: Bearer YOUR_CLIENT_ACCESS_TOKEN'
+```
+
+Have a look at the [test dataset](/docs/test-dataset) for more sample passholders or events.
+
+Example response:
+
+```http
+HTTP/1.1 200 OK
+Content-Type: application/json
+
+{
+  "available": [
+    {
+      "id": "SOCIALTARIFF",
+      "name": "Kansentarief",
+      "price": 2,
+      "type": "SOCIALTARIFF",
+      "remaining": 1
+    },
+    {
+      "id": "COUPON_1079",
+      "name": "[TEST] €1 voor een evenement voor tieners (GENT)",
```

Comment:
Won't the `[TEST]` in these names be confusing? Everything on the test environment is test data, and the square brackets might make it look like a technical thing that needs to be parsed?